libraries/standard/http/cbmc/proofs/Makefile-project-defines

```diff
@@ -27,3 +27,7 @@ INCLUDES += -I$(SRCDIR)/platform/include
 
 # Preprocessor definitions -D...
 DEFINES += -Dhttp_EXPORTS
+
+# Ensure that all assumptions are sound by checking that "all pointers
+# in pointer primitives are valid or null"
+CHECKFLAGS += --pointer-primitive-check

```

Comment:
Shouldn't this be in makefile.common?

Reply:
Per offline conversation we are using it here first but cbmc team wishes not to break other projects.